test/test_transformers.py

```diff
@@ -656,6 +656,23 @@ def test_mha_native_args(self, nb_heads, bias):
 
             mha(query=x, key=x, value=x, key_padding_mask=pad_mask)
 
+    def test_kpm_mask_trailing_column_with_nested_tensor(self):
```

Comment:
Make sure to add a `device` arg to this test and use it in the constructor of the x, and mask accordingly on rebase

Reply:
Done.